tesseract_motion_planners/trajopt_ifopt/include/tesseract_motion_planners/trajopt_ifopt/trajopt_ifopt_problem.h

```diff
@@ -67,6 +73,11 @@ struct TrajOptIfoptProblem
 
   std::vector<std::shared_ptr<trajopt_sqp::SQPCallback>> callbacks;
 
+  /** @brief The OSQP convex solver settings to use */
+  std::unique_ptr<OsqpEigen::Settings> convex_solver_settings{ nullptr };
+
+  std::shared_ptr<trajopt_sqp::QPSolver> qp_solver;
```

Comment:
@rjoomen Is there a specific reason you want to store this in the problem because it does not seem necessary?